DatabaseTemplate/Modules/create-database-template.psm1

```diff
@@ -512,15 +512,29 @@ function New-DatabaseTemplateNuspec {
     
     $nuspecPath = Join-Path $populatedTemplatePath "$packageNuspecName.nuspec"
 
+    $Id = '$id$'
```

Comment:
I wasnt able to get an idea for the reason behind this change from your commits, so could you explain to me what this change was for?

Reply:
The nuspec file used to only allow for defaults that had to be replaced in the call to the nuget pack function.  Now that we are sharing the pack method with build.githubactions I placed the optional setting of these properties in the ps script while retaining the default behavior of nuspec file properties being set to placeholder values